src/components/network/network-table.js

```diff
@@ -8,34 +8,49 @@
 /**
  * NetworkTable renders given multi-dimensional data as a table.
  * @param {!Object} data Data object of the view.
+ * @extends {genotet.ViewLoader}
```

Comment:
It's not reasonable to extend the class just to use one method. You can add a signal method under the table.